management-core/mysql/data.sql

```diff
@@ -0,0 +1,31 @@
+use ManagementProduct;
```

Comment:
What happens if the DB doesn't exist?

Reply:
Here I missed add an entry to README.md, because if you run docker-compose up -d it will create the data base(if not exists), tables and Stored Procedures if not exists, I do not know if this is the standard way, any suggestion will be appreciated